src/PrintOrder/PrintOrder.js

```diff
@@ -37,7 +37,7 @@ export const PrintOrderComponent = ({ mutator, order, orderLine, onCancel }) =>
       const { compositePoLines } = order;
       const linesToPrint = orderLine ? [orderLine] : compositePoLines;
       const printData = compositePoLines?.length
-        ? { lines: await getExportData(mutator, linesToPrint, [order], intl) }
+        ? { lines: await getExportData(mutator, linesToPrint, [order], [], intl) }
```

Comment:
so we don't support custom fields in this print template? (I don't know requirements for custom fields that's why I ask)

Reply:
Correct. A requirement was only defined for csv export ([UIOR-1231](https://folio-org.atlassian.net/browse/UIOR-1231)).

Reply:
thanks, I wanted to put empty array as well, but wasn't sure about requirements so redirected this ticket to your team :)